inst/extdata/PA2022CH_fr_exec_summary/template.Rmd

```diff
@@ -235,7 +235,7 @@ tryCatch(
 
 ```{=latex}
 
-\* Technologies «à faible émission de CO\textsubscript{2}» (low-carbon): énergies renouvelables pour l'électricité et énergie électrique pour l'automobile. Technologies «à forte émission de carbone» (high-carbon): charbon, pétrole, gaz 
+** Technologies \guillemetleft à faible émission de CO\textsubscript{2}\guillemetright  (low-carbon): énergies renouvelables pour l'électricité et énergie électrique pour l'automobile. Technologies \guillemetleft à forte émission de carbone» (high-carbon): charbon, pétrole, gaz 
```

Comment:
last `\guillemetleft` has a space after it, and no apparent closing `\guillemetright`

Reply:
spacing fixed